packages/react/src/components/ComposedModal/ComposedModal.jsx

```diff
@@ -112,6 +114,7 @@ class ComposedModal extends React.Component {
     header: {},
     iconDescription: 'Close',
     passiveModal: false,
+    testID: 'ComposedModal',
```

Comment:
Please add this to the main wrapper component as well.

Reply:
Done!